### ISSUE TITLE
Table with one element does not show any rows

### ISSUE THREAD

**Issue by mobislan** [esvit/ng-table#190]
I forked your checkbox example to have only one element.
I think you call the $watch $params wrongly.

http://plnkr.co/edit/F9BmHCeYAynSivp3C2tK

**Comment by iyel** [comment]
it is fixed in v0.5.0
![screen shot 2015-02-18 at 11 19 34 pm](https://cloud.githubusercontent.com/assets/929834/6256906/a6cf1d82-b7c4-11e4-81a8-57be913f26b2.png)

**Comment by egamonal** [comment]
maybe we're doing something wrong? it might be related to pagination. if I set `count: 10`, it works.

```
$scope.surveyStats = new ngTableParams({
                page: 1,            // show first page
                count: surveyStats.length           // count per page
            }, {
                counts: [],
                total: surveyStats.length, 
                getData: function ($defer, params) {
                    console.log("in GETDATA", surveyStats );
                    $defer.resolve(surveyStats);
                }
            });
```
it never enters in `getData` with these parameters and an endpoint in an API returning an array of 1 element, like this:
```
http :8000/v2.0/survey-stats/ "Authorization:Bearer 542844...c09161" 
HTTP/1.0 200 OK
Allow: GET, HEAD, OPTIONS
Content-Type: application/json
Date: Wed, 18 Feb 2015 20:21:24 GMT
Last-Modified: Wed, 18 Feb 2015 19:21:24 GMT
Server: WSGIServer/0.1 Python/2.7.3
Vary: Accept
X-Frame-Options: SAMEORIGIN

[
    {
        "count": 689, 
        "survey": 21, 
        "title": "a title"
    }, 
    {}
]
```
